Patches/Mechanized Armor Set/Ammo_&_Apparel_Defs.xml

```diff
@@ -0,0 +1,139 @@
+					<xpath>Defs/ThingDef[defName="Mechanized_Helm_Apparel"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<ArmorRating_Sharp>3.5</ArmorRating_Sharp>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Mechanized_Helm_Apparel"]/statBases/ArmorRating_Blunt</xpath>
+					<value>
+						<ArmorRating_Blunt>5.2</ArmorRating_Blunt>
+					</value>
+				</li>
+
+				
+				<!--Plate Armour-->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Mechanized_Plate_Apparel"]/statBases</xpath>
+					<value>
+						<Bulk>100</Bulk>
+						<WornBulk>0</WornBulk>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Mechanized_Plate_Apparel"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<ArmorRating_Sharp>7.2</ArmorRating_Sharp>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Mechanized_Plate_Apparel"]/statBases/ArmorRating_Blunt</xpath>
+					<value>
+						<ArmorRating_Blunt>5.5</ArmorRating_Blunt>
+					</value>
+				</li>
+				
+				<!--Musket-->
+				<!--Tools-->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Mechanized_Musket"]/tools</xpath>
+					<value>
+						<tools>
+							<li Class="CombatExtended.ToolCE">
+								<label>Stock</label>
+								<capacities>
+									<li>Blunt</li>
+								</capacities>
+								<power>8</power>
+								<cooldownTime>1.55</cooldownTime>
+								<chanceFactor>1.5</chanceFactor>
+								<armorPenetrationBlunt>2.755</armorPenetrationBlunt>
+								<linkedBodyPartsGroup>Stock</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+								<label>Barrel</label>
+								<capacities>
+									<li>Blunt</li>
+									<li>Poke</li>
+								</capacities>
+								<power>9</power>
+								<cooldownTime>2.0</cooldownTime>
+								<armorPenetrationBlunt>1.630</armorPenetrationBlunt>
+								<linkedBodyPartsGroup>Barrel</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+								<label>Bayonet</label>
+								<capacities>
+									<li>Stab</li>
+								</capacities>
+								<power>23</power>
+								<cooldownTime>2.6</cooldownTime>
+								<armorPenetrationSharp>4.755</armorPenetrationSharp>
+								<linkedBodyPartsGroup>Muzzle</linkedBodyPartsGroup>
+							</li>	
+							</tools>	
+							</value>
+							</li>
+								<!--Shooting-->						
+								<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+									<defName>Mechanized_Musket</defName>
+									<statBases>
+										<WorkToMake>40000</WorkToMake>
+										<Mass>1.25</Mass>
+										<Bulk>3.90</Bulk>
+										<SwayFactor>1.02</SwayFactor>
+										<ShotSpread>0.36</ShotSpread>
+										<SightsEfficiency>0.9</SightsEfficiency>
+										<RangedWeapon_Cooldown>0.85</RangedWeapon_Cooldown>
+									</statBases>
+									<costList>

```

Comment:
You can remove this `<costList>` part, as it's redundant. If it's not included in the operation, it's simply left alone.